doc/source/conf.py

```diff
@@ -375,11 +375,16 @@
    import os
    os.chdir(r'{os.path.dirname(os.path.dirname(__file__))}')
 """
+footer = f"""<p class="copyright">
+    &copy{datetime.now().year}
+    pandas via <a href="https://numfocus.org">NumFOCUS, Inc.</a>
+    Hosted by <a href="https://www.ovhcloud.com">OVHcloud</a>.</p>"""
 
 
 html_context = {
     "redirects": dict(moved_api_pages),
     "header": header,
+    "footer": footer
```

Comment:
What you want is to edit `html_theme_options`, and specify the footer html there. My understanding is that in older pydata sphinx theme versions we could only append to the predefined footer, and now we can fully overwrite it. That's what we want to do, to have the same exact footer as now, without having to have our custom footer template.

Reply:
I've been looking for a solution to this, I'm still working on it.